### PR TITLE
Changed affiliation of Arne Nordmann to none/private

### PR DESCRIPTION
**Pull request by norro**
Signed-off-by: Arne Nordmann <github@norro.de>

### CODE REVIEW

**Comment by ralph-lange** [comment]
Note that there is a setup.py, which needs also to be updated (just search the whole repo for your old e-mail address.). Not sure whether two maintainers can be listed there. If not, then use your name and e-mail as first maintainer, please.

File: NOTICE [hunk 1]
```diff
@@ -30,3 +30,6 @@ Robert Bosch GmbH
 
 BSH Hausgeräte GmbH
     Thilak Raj <chikmagalore.thilak@bshg.com>
+
+none
+    Arne Nordmann <github@norro.de>
```
Comment (ralph-lange) [inline NOTICE]:
```suggestion
Arne Nordmann <github@norro.de>
```
No need for 'none'.

**Review by ralph-lange — CHANGES_REQUESTED** [review]

**Comment by norro** [comment]
There doesn't seem to be an agreed way how to add multiple maintainers to setup.py that is not confusing pypi and the like. So I just went with my name there.

**Review by ralph-lange — APPROVED** [review]
LGTM

**Pull request merged**